test/helpers/MathUtils.js

```diff
@@ -37,6 +37,16 @@ function roundDiv(a, b) {
     return a.div(b).toFixed(0, Decimal.ROUND_HALF_UP);
 }
 
+function mulDivF(a, b, c) {
```

Comment:
Please use the `const mulDiv = (..) =>` notation, like we do in the rest of the code (other than some legacy convention here... and there).

Reply:
Done (the rest of this file in fact doesn't use `const`).

Reply:
It seems that the `arguments` keyword can be used inside a function, but when referred to from a function pointer (i.e., a "const" declaration of a variable which points to a function), it takes an additional item, which is the path of the file in which it is declared.
Subsequently, the function throws an exception (attempting to convert the path string to a `Decimal` object).
Will revert this change and handle it in a separate task.

Reply:
See my comment re: `const protectedAmountPlusFee = (...args) => {`

Reply:
Do you mean, use:
```
const decimalize = (func, ...args) => func(...args.map(x => Decimal(x))));
```
?

Reply:
Yeah